ingest/example/bulk_pretty

```diff
@@ -0,0 +1,27 @@
+{  
+  "index":{  
+    "_id":"1",
+    "_index":"dtdi",
+    "_type":"mineral-observation"
+  }
+}{  
+  "locality":[  
+    {  
+      "country":"Canada",
+      "region":"Québec, Canada",
+      "mine":"Poudrette quarry (Demix quarry; Uni-Mix quarry; Desourdy quarry; Carrière Mont Saint-Hilaire), Mont Saint-Hilaire, La Vallée-du-Richelieu RCM, Montérégie, Québec, Canada",
+      "coordinates":{  
+        "lat":"45.56277778",
+        "lon":"-73.14166667"
+      },
+      "mindat":"http://www.mindat.org/loc-599.html"
+    }
+  ],
+  "description":"Named after the Abenaki, the indigenous people who inhabited the area around Mont Saint-Hilaire in southern Québec, prior to the arrival of European settlers.",
+  "mindat":"http://www.mindat.org/min-2.html",
+  "mineral":{  
+    "name":"Abenakiite-(Ce)",
+    "rruffchemistry":"Na_26_Ce^3+^_6_(SiO_3_)_6_(PO_4_)_6_(CO_3_)_6_(S^4+^O_2_)O",
```

Comment:
Is there a standard syntax for encoding this chemistry in text?

I see ``Na26Ce6(Si6O18)(PO4)6(CO3)6(SO3)`` at the the mandate link.